packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -819,11 +819,11 @@ export interface IIntervalCollection<TInterval extends ISerializableInterval>
 	/**
 	 * Changes the endpoints of an existing interval.
 	 * @param id - Id of the interval to change
-	 * @param start - New start value, if defined. `undefined` signifies this endpoint should be left unchanged.
-	 * @param end - New end value, if defined. `undefined` signifies this endpoint should be left unchanged.
+	 * @param start - New start value.
```

Comment:
Should we also note here that the current start/end positions can be passed in to avoid changing an endpoint?